src/main/java/com/fasterxml/uuid/impl/TimeBasedEpochGenerator.java

```diff
@@ -97,13 +97,16 @@ public TimeBasedEpochGenerator(Random rnd, UUIDClock clock)
     /* UUID generation
     /**********************************************************************
      */
-    
+
     @Override
-    public UUID generate()
+    public UUID generate() {
+        return generate(_clock.currentTimeMillis());
+    }
+    
+    public UUID generate(long rawTimestamp)
```

Comment:
Let's rename this as `construct()`, since it's not fully generating any more.

Also add Javadoc with `@since 4.3` to indicate addition.